src/Pact/Native.hs

```diff
@@ -735,6 +742,12 @@ makeList g i [TLitInteger len,value] = case typeof value of
   Left ty -> evalError' i $ "make-list: invalid value type: " <> pretty ty
 makeList _ i as = argsError i as
 
+-- TODO: Should be there limits on the bounds of the enumerated sequence?
```

Comment:
Remove comment. To answer your question see https://github.com/kadena-io/pact/blob/4971ab6078b75eb612d83d56f1e7cd139a5a2ba8/src/Pact/Gas/Table.hs#L201 

in which if the list exceeds a certain length it starts to be penalized exponentially in gas costs.